google Oauth/config.py

```diff
@@ -0,0 +1,3 @@
+GOOGLE_API_CLIENT_ID = '595445376676-2novv7in7s4knmg9243i0h20pq3pqrui.apps.googleusercontent.com'
```

Comment:
Please use a .env file for sensitive information such as this. Then you can call the variable from the .env file. Please do not forget to put the .env into the gitignore file